examples/adeo-kafka-request-reply.yml

```diff
@@ -0,0 +1,331 @@
+asyncapi: 3.0.0
+info:
+  title: Adeo AsyncAPI Case Study
+  version: "%REPLACED_BY_MAVEN%"
+  description: >
+    This Adeo specification illustrates how ADEO uses AsyncAPI to document some of their exchanges
+  contact:
+    name: AsyncAPI team
+    email: case-study@asyncapi.com
```

Comment:
Just a heads up this email doesn't exist. Maybe we should use `info@asyncapi.io` instead.

Reply:
fixed